lib/exe/css.php

```diff
@@ -344,6 +360,37 @@ function css_fixreplacementurls($replacements, $location) {
 }
 
 /**
+ * Wrapper for the files, content and mediatype for the event CSS_STYLES_INCLUDED
+ *
+ * @param string $mediatype type ofthe current media files/content set
+ * @param array $files set of files that define the current mediatype
+ * @return array
+ */
+function css_filewrapper($mediatype, $files=array()){
+    return array(
+            'files'                 => $files,
+	        'mediatype'             => $mediatype,
+	        'encapsulate'           => in_array($mediatype, array('screen', 'print')),
```

Comment:
See #1131 - the whitelist should be a lot broader. There is indeed a bit of piggybacking. This function is a container to the Event that allows developers to inject/remove CSS depending on the context or state (e.g. admin is logged in -> admin styles) and allow removal of the currently hardcoded DW core styles. This is a new functionality - but it should ensure backwards compatibility as well. the encapsulate check was meant to ensure that only real media types will be encapsulated. Everything else should go into all or screen - while "all" should not be encapsulated. I had situations when a large ``@media all {}`` section had problems with CSS parsers.